src/cli/info.rs

```diff
@@ -20,8 +27,14 @@ impl Options {
         let backends = BACKENDS.deref();
 
         if self.json {
-            println!("{}", serde_json::to_string_pretty(&backends)?);
+            let info = Info {
+                version: env!("CARGO_PKG_VERSION"),
+                backends,
+            };
+
+            println!("{}", serde_json::to_string_pretty(&info)?);
         } else {
+            println!("Version {} starting up", env!("CARGO_PKG_VERSION"));
             for backend in backends {
                 println!("Backend: {}", backend.name());
```

Comment:
You could implement `Display` also for `Info` and use it in both paths.

Reply:
I shall look into that.